### PR TITLE
Frontend sturcturing

### PR DESCRIPTION
**Pull request by csullivan710**
Completed structuring, except the styling. This was made into a new card to be completed later.

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react'
+
+
+class Footer extends Component{
+  render(){
+    return(
+      <footer>
+
```
Comment (sjproctor) [inline src/components/Footer.js:8]: It would be a good idea to make sure this component is actually working.

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react'
+
+
+class Header extends Component {
+  render() {
+    return (
+      <>
+
```
Comment (sjproctor) [inline src/components/Header.js:8]: It would be a good idea to make sure this component is actually working by rendering some content.

**Review by sjproctor — APPROVED** [review]
Take a sec and add some tags to be rendered to your Header and Footer. Then you are good.

File: src/components/Footer.js (hunk 1 above)
Comment (csullivan710) [reply src/components/Footer.js:8]: Tags added

File: src/components/Header.js (hunk 2 above)
Comment (csullivan710) [reply src/components/Header.js:8]: We discussed it, but got stuck with our typo. Thanks for the headsup!

**Pull request merged**